### DISCUSSION TITLE
Is there a way to view all the cards (reviewed and unreviewed)

### DISCUSSION THREAD

**Discussion by sameerad2001** [st3v3nmw/obsidian-spaced-repetition#386] [Q&A] [answered]
TLDR : Is there a way to re view the flash cards **after going through them once** or just view all the cards i.e reviewed and unreviewed

I am having a little bit of trouble formatting my questions and answers so that they show up on the cards properly and I have taken to using the format below (because it is easier to convert some of my older notes to this format)
The problem is some of the cards do not display properly and It would be helpful to _view the changes I make without having to constantly remove the tags_ ( Eg :  `<!--SR:!2022-01-26,1,270-->` )
This could also be helpful in reviewing the notes in the form of cards which have concise hints instead of reading the entire thing

### Format :
----
#flashcards/topic

### Question
?
```
  Multiline hint/summary (shows up on the card as the answer)
  
  ..... hint ends here
```  

- (There is a white line above)
- Not a part of the card
- Long and detailed answer
- .....

----

**Comment by ronzulu** [comment, marked as answer]
Hi  @sameerad2001 

Do you know about the "cram" features available from obsidian commands:

![image](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/75528127/c97ff3ae-c55b-41f1-b1b6-0b69cf7e2c41)

Is that what you are after?

Cheers
Ronny

**Reply by dnnane** [reply]
Fantastic!

**Reply by ClayLaMothe** [reply]
New Spaced Repetition user here. This was exactly what I was looking for, thanks ronzulu (and OP for asking).

**Comment by ronzulu** [comment]
Hi @sameerad2001 does my post above answer your question completely?

If so, please mark this as answered.

If not, please let us know what specifically is unclear.

Cheers
Ronny

**Reply by sameerad2001** [reply]
Sorry for taking so long to mark this question as answered. I haven't been using this plugin for a while and it slipped my mind to check.
Thank you for answering.
The cram feature seems to be new and I will try to check it out when I can.
